tests/unit/parsec/test_upgrade.py

```diff
@@ -48,7 +48,9 @@ def test_simple():
     upg.deprecate('1.3', ['section A'], ['Heading A'])
     # NOTE change to new item keys here!
     upg.deprecate('1.3', ['Heading A', 'cde'], ['Heading A', 'CDE'])
-    upg.deprecate('1.4', ['Heading A', 'abc'], cvtr=x2, silent=True)
+    upg.deprecate(
+        '1.4', ['Heading A', 'abc'], ['Heading A', 'abc'], cvtr=x2,
+        silent=True)
```

Comment:
Can you add a test to demonstate the new behaviour as you've used it - i.e. `upg['new'] = None`?

Reply:
This should do the trick: https://github.com/cylc/cylc-flow/pull/3914/commits/4e423427e4d346af048c6b78dc8c9a4e81aba290